src/inline_table.rs

```diff
@@ -93,6 +93,46 @@ impl InlineTable {
         }
     }
 
+    /// Sort Key/Value Pairs of the table using the using the comparison function `compare`.
+    ///
+    /// The comparison function receives two key and value pairs to compare (you can sort by keys or
+    /// values or their combination as needed).
+    pub fn sort_by<F>(&mut self, compare: F)
```

Comment:
Could you rename the functions to `sort_values_by` for consistency?

Reply:
Done!